MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -458,22 +458,25 @@ - (MSALUIBehavior)uiBehavior
     @throw @"Do not recognize prompt behavior";
 }
 
-- (void)acquireTokenInteractive:(id)sender
+- (NSString *)currentClientId
 {
-    (void)sender;
-    MSALTestAppSettings *settings = [MSALTestAppSettings settings];
-    MSALAuthority *authority = [settings authority];
-    NSString *clientId;
-    NSDictionary *extraQueryParameters = [NSDictionary msidDictionaryFromWWWFormURLEncodedString:_extraQueryParamsField.text];
-    
-    if([[MSALTestAppAuthorityViewController currentTitle] containsString:@"/tfp/"])
+    if ([[MSALTestAppAuthorityViewController currentTitle] containsString:@"/tfp/"])
     {
-        clientId = B2C_TEST_APP_CLIENT_ID;;
+        return B2C_TEST_APP_CLIENT_ID;
     }
     else
     {
-        clientId = TEST_APP_CLIENT_ID;
+        return TEST_APP_CLIENT_ID;
     }
+}
+
+- (void)acquireTokenInteractive:(id)sender
```

Comment:
we can address this separately :) Normally IBActions should have a sender. But it's true that those are not IBActions. Anyway, won't change it as part of this PR, otherwise I'll want to rebuild the test app :)